### ISSUE TITLE
[BUG] Windows 10 desktop cannot install

### ISSUE THREAD

**Issue by itsJoKr** [leoafarias/fvm#212]
First I tried to install the desktop version. When running the dart.exe file it will immediately exit. I tried to run it with a command prompt or PowerShell and that would return
`Unable to find DartDev snapshot.`.

FVM CLI is installed and working.

Pretty fresh Windows 10
Poweshell 5.1.19041.610
Installed Git, Dart, Flutter
AMD Ryzen 5

**Comment by jans-y** [comment]
Just add ref of the commit to the file_chooser dependency inside `pubspec.yaml`
```
  file_chooser:
    git:
      url: git://github.com/google/flutter-desktop-embedding.git
      path: plugins/file_chooser
      ref: 43901ebd12548a3c2a077d4dbeb3b56f0a0bdd09
```

**Comment by leoafarias** [comment]
This has been fixed by #211
